package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -35,45 +39,56 @@
 Water bridge is defined as a bridging water which simutaneously form
 two hydrogen bonds with atoms from both selection 1 and selection 2.
 
-A water bridge can form between two hydorgen bond acceptors.
+A water bridge can form between two hydrogen bond acceptors.
 
 e.g. -CO\ :sub:`2`\ :sup:`-`:···H−O−H···:\ :sup:`-`\ O\ :sub:`2`\ C-
 
-A water bridge can also form between two hydorgen bond donors.
+A water bridge can also form between two hydrogen bond donors.
 
 e.g. -NH···:O:···HN- (where O is the oxygen of a bridging water)
 
-A hydorgen bond acceptor and another hydorgen bond donor can be bridged by a water.
+A hydrogen bond acceptor and another hydrogen bond donor can be bridged by a
+water.
 
 e.g. -CO\ :sub:`2`\ :sup:`-`:···H−O:···HN- (where H−O is part of **H−O**\ −H)
 
-The :class:`WaterBridgeAnalysis` class is modeled after the  \ :class:`~MDAnalysis.analysis.hbonds.hbond_analysis.HydrogenBondAnalysis`.
+The :class:`WaterBridgeAnalysis` class is modeled after the  \
+:class:`~MDAnalysis.analysis.hbonds.hbond_analysis.HydrogenBondAnalysis`.
 
-Options:
-  - *water_selection* (``resname SOL``): the selection string for the bridging water
-  - donor-acceptor *distance* (Å): 3.0
-  - Angle *cutoff* (degrees): 120.0
-  - *forcefield* to switch between default values for different force fields
-  - *donors* and *acceptors* atom types (to add additional atom names)
+The following keyword arguments are important to control the behavior of the
+water bridge analysis:
+
+ - *water_selection* (``resname SOL``): the selection string for the bridging
+   water
+ - donor-acceptor *distance* (Å): 3.0
+ - Angle *cutoff* (degrees): 120.0
+ - *forcefield* to switch between default values for different force fields
+ - *donors* and *acceptors* atom types (to add additional atom names)
 
 .. _Analysis Output:
 
 Output
 ------
 
-The results are a list of hydrogen bonds between the selection 1 or selection 2 and the bridging water.
+The results are a list of hydrogen bonds between the selection 1 or selection 2
+and the bridging water.
+
+Each list is formated similair to the \ :attr:`HydrogenBondAnalysis.timeseries
```

Comment:
similar

(run a spell checker, especially when writing lots of docs)

Reply:
@orbeckst Sorry, I have a spell check integrated in word but I guess I probably need a spell check integrated into the IDE.